compiler/src/dmd/funcsem.d

```diff
@@ -1217,3 +1217,152 @@ extern (D) bool checkForwardRef(FuncDeclaration fd, const ref Loc loc)
     }
     return false;
 }
+
+/*************************************************
+ * Find index of function in vtbl[0..length] that
+ * this function overrides.
+ * Prefer an exact match to a covariant one.
+ * Params:
+ *      vtbl     = vtable to use
+ * Returns:
+ *      -1      didn't find one
+ *      -2      can't determine because of forward references
+ */
+int findVtblIndex(FuncDeclaration fd, Dsymbol[] vtbl)
```

Comment:
Need to also update the documentation, hence the doc tester failure:

`1msrc/dmd/funcsem.d(1231): [1;33mWarning: [mDdoc: parameter count mismatch, expected 2, got 1`

Reply:
updated